lib/linguist/languages.yml

```diff
@@ -330,6 +330,15 @@ BitBake:
   - .bb
   ace_mode: text
 
+Blade:
+  type: markup
+  group: HTML
+  extensions:
+    - .blade
+    - .blade.php
+  tm_scope: text.html.blade.php
```

Comment:
This needs to be `text.html.php.blade`. Wherever the scope is mentioned/listed, it needs to match what's listed in the [grammar file](https://github.com/jawee/language-blade/blob/master/grammars/blade.cson#L1). Verbatim.